src/utils/pluginDriver.ts

```diff
@@ -361,3 +395,11 @@ const uncacheablePlugin: (pluginName: string) => PluginCache = pluginName => ({
 		return false;
 	}
 });
+
+function hookDeprecationWarning(name: string, newName: string, plugin: Plugin, pidx: number) {
+	return {
+		code: name.toUpperCase() + '_HOOK_DEPRECATED',
+		message: `The ${name} hook used by plugin ${plugin.name ||
+			`at position ${pidx + 1}`} is deprecated. The ${newName} hook should be used instead.`
+	};
+}
```

Comment:
Might be nice to add a `.url` with a link to the upcoming plugin API section on the website once it is available.

Reply:
Similarly, we could add a link to the big list of options when an unknown option is passed: https://rollupjs.org/guide/en#big-list-of-options

Reply:
👍